rx_signal_processing/dsp.cu

```diff
@@ -182,13 +255,13 @@ namespace {
       total_beam_dirs += beam_count;
     }
 
-    std::vector<cuComplex> beamformed_samples(dp->get_num_antennas() * 
-                                                total_beam_dirs * 
-                                                num_samples_after_dropping);
+    std::vector<cuComplex> beamformed_samples_main(total_beam_dirs * num_samples_after_dropping);
+    std::vector<cuComplex> beamformed_samples_intf(total_beam_dirs * num_samples_after_dropping);
 
     auto beam_phases = dp->get_beam_phases();
-    beamform_samples(output_samples, beamformed_samples, beam_phases, 
-                      dp->get_num_antennas(), dp->get_beam_direction_counts(), 
+    beamform_samples(output_samples, beamformed_samples_main, beamformed_samples_intf, beam_phases, 
+                      dp->sig_options.get_main_antenna_count(), 
+                      dp->sig_options.get_main_antenna_count(), dp->get_beam_direction_counts(), 
```

Comment:
should be get_intf_antenna_count I think?

Reply:
Yup. Auto complete failed me there.